python/hail/representation/annotations.py

```diff
@@ -70,3 +71,29 @@ def get(self, item, default=None):
         :returns: Value of item if found, or default value if not.
         """
         return self._attrs.get(item, default)
+
+
+@typecheck(struct=Struct)
+def to_dict(struct):
```

Comment:
I'm not totally sure. The whole design is a bit weird and regrettable, but I did it to provide the ability to select stuff with dots: `vds.globals.foo.bar`. I only use to_dict in the pprint monkey patch at the moment.